schemas/questionnaire_v1.json

```diff
@@ -246,6 +246,26 @@
                     },
                     "empty_list_text": {
                       "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+                    },
+                    "related_answers": {
+                      "type": "array",
+                      "minItems": 1,
+                      "items": {
+                        "$ref": "https://eq.ons.gov.uk/value_sources.json#/answer_value_source"
+                      },
+                      "description": "Ids of answers to be displayed on list summary"
+                    },
+                    "item_anchor_answer_id": {
+                      "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+                      "description": "The main answer_id for the item title used as anchor in links on list summary"
+                    },
+                    "item_label": {
+                      "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders",
+                      "description": "The accompanying label for the item title on list summary"
```

Comment:
Worth adding `"examples": ["Name of company", "Full name"]`

Reply:
Added examples, thanks!